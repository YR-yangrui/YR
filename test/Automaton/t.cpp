/*******************************
Author:galaxy yr
LANG:C++
Created Time:2019年06月09日 星期日 21时11分30秒
*******************************/
#include<iostream>
#include<cstdio>
using namespace std;
int main()
{
        freopen("t.in","r",stdin);
        freopen("t.out","w",stdout);
        int * a,*b;
        b=new int ;
        *b=3;
        a=b;
        cout<<a<<' '<<b<<endl;
        cout<<*a<<' '<<*b;
}
