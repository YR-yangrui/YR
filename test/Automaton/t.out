0x564c3c0b8e70 0x564c3c0b8e70
3 3